python/az/aro/azext_aro/tests/latest/integration/test_aro_scenario.py

```diff
@@ -30,6 +30,10 @@ def test_aro(self, resource_group):
             self.check('tags.foo', 'boo')
         ])
 
+        self.cmd('az aro get-versions --location eastus', checks=[
+            self.check('4.10.20')
+        ])
+
```

Comment:
While this test will work now, with the hard coded "4.10.20" return value coming from the RP, that is going to very shortly be switched to dynamically loading from the CosmoDB OpenShiftVersions database. When that happens, this test will fail. In short, since we know it will be redundant very shortly, it's better to leave it out.

Reply:
Thanks David, have updated with a commit to remove the hard coded test.